### PR TITLE
Feature/Orders Autoreload

### PR DESCRIPTION
**Pull request by rishabh-vinsol**
orders

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -50,6 +50,7 @@ def order_success
       @order.received unless @order.placed_on
       @order.send_confirmation_email
       @payment.update(status: payment_status, mode: payment_type)
+      ActionCable.server.broadcast('order_success', @order.details)
```
Comment (himanshumishra31) [inline app/controllers/orders_controller.rb:53]: Write in order callback when status is changed

Comment (rishabh-vinsol) [reply app/controllers/orders_controller.rb]: Done

File: app/models/order.rb [hunk 2]
```diff
@@ -26,6 +26,7 @@ class Order < ApplicationRecord
 
   before_save :check_placed_on, if: %i[cancelled? status_changed?]
   after_commit :send_confirmation_email, if: :received?
+  after_update_commit :broadcast_order, if: :received?
```
Comment (himanshumishra31) [inline app/models/order.rb:29]: fix all the callbacks conditions as discussed on call

Comment (rishabh-vinsol) [reply app/models/order.rb]: Ok sir

**Pull request merged**